opentutor_classifier/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool]
+[tool.poetry]
+name = "opentutor_classifier"
+version = "1.0.0"
+description="train and run inference for open tutor"
+authors=[]
+
+[tool.poetry.dependencies]
+# python = "^3.8"
+python = ">=3.8,<3.10"
+celery = "==5.0.5"
```

Comment:
can we try updating all of our pyproject.toml deps to allow `poetry update` to find compatible versions?

In node, we use carot versions, e.g. ^4.0.1, which means "any non-major-version update, e.g. ">= 4.0.1 < 5"

Looks like carot uses the same semver spec:

https://python-poetry.org/docs/dependency-specification/#caret-requirements

...can we try converting all our versions here to carot version, run `make update` or whatever is the command and then see if tests still pass? Will be interesting too to  look at the diff of the poetry-lock file